api/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # This must be run in project root, to pick up dataset_config/.
-FROM python:2
+FROM python:3
```

Comment:
I see one bug with local 1000 Genomes. Before this PR:

![8x2a0GLunbD](https://user-images.githubusercontent.com/10929390/67602397-54362600-f72b-11e9-8b6c-6c2b125aa41c.png)

After:

![tebTHhYsf3D](https://user-images.githubusercontent.com/10929390/67602412-5f895180-f72b-11e9-93db-db58c92cce97.png)

Could a [unit test](https://github.com/DataBiosphere/data-explorer/blob/master/api/data_explorer/test/test_elasticsearch_util.py) have caught this?

Reply:
I believe so. I will fix this, and right a unit test for it in the new PR